pg-extend/src/lib.rs

```diff
@@ -60,21 +63,38 @@ macro_rules! pg_magic {
     };
 }
 
-/// Returns the slice of Datums, and a parallel slice which specifies if the Datum passed in is (SQL) NULL
+#[cfg(feature = "postgres-12")]
+type NullableDatum = pg_sys::NullableDatum;
+
+// PostgreSQL 11 and earlier don't have this type; declare it locally.
+#[cfg(not(feature = "postgres-12"))]
+struct NullableDatum {
+    value: Datum,
+    isnull: bool,
+}
```

Comment:
Do you think this should go into `pg_datum.rs` instead?

Reply:
Oh, I thought we did have this? But yes it should go there.

Reply:
You probably want to use this: https://github.com/bluejekyll/pg-extend-rs/blob/master/pg-extend/src/pg_datum.rs#L285

Reply:
Ah yes, that makes lots of sense. Haven't read Rust code for some time so I was confused at first.